cmd/fyne_demo/screens/container.go

```diff
@@ -0,0 +1,119 @@
+package screens
+
+import (
+	"fmt"
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// ContainerScreen loads a tab panel for containers and layouts
+func ContainerScreen() fyne.CanvasObject {
+	return widget.NewTabContainer(
+		widget.NewTabItem("Split", makeSplitTab()),
+		widget.NewTabItem("Scroll", makeScrollTab()),
+		// layouts
+		widget.NewTabItem("Border", makeBorderLayout()),
+		widget.NewTabItem("Box", makeBoxLayout()),
+		widget.NewTabItem("Center", makeCenterLayout()),
+		widget.NewTabItem("Grid", makeGridLayout()),
+	)
+}
+
+func makeBorderLayout() *fyne.Container {
+	top := makeCell()
+	bottom := makeCell()
+	left := makeCell()
+	right := makeCell()
+	middle := widget.NewLabelWithStyle("BorderLayout", fyne.TextAlignCenter, fyne.TextStyle{})
+
+	borderLayout := layout.NewBorderLayout(top, bottom, left, right)
+	return fyne.NewContainerWithLayout(borderLayout,
+		top, bottom, left, right, middle)
+}
+
+func makeBoxLayout() *fyne.Container {
+	top := makeCell()
+	bottom := makeCell()
+	middle := widget.NewLabel("BoxLayout")
+	center := makeCell()
+	right := makeCell()
+
+	col := fyne.NewContainerWithLayout(layout.NewVBoxLayout(),
+		top, middle, bottom)
+
+	return fyne.NewContainerWithLayout(layout.NewHBoxLayout(),
+		col, center, right)
+}
+
+func makeButtonList(count int) []fyne.CanvasObject {
+	var items []fyne.CanvasObject
+	for i := 1; i <= count; i++ {
+		index := i // capture
+		items = append(items, widget.NewButton(fmt.Sprintf("Button %d", index), func() {
+			fmt.Println("Tapped", index)
+		}))
+	}
+
+	return items
+}
+
+func makeCell() fyne.CanvasObject {
+	rect := canvas.NewRectangle(&color.RGBA{128, 128, 128, 255})
+	rect.SetMinSize(fyne.NewSize(30, 30))
+	return rect
+}
+
+func makeCenterLayout() *fyne.Container {
+	middle := widget.NewButton("CenterLayout", func() {})
```

Comment:
nit: to be consistent this should be a label like the other ones

Reply:
If it were a label you would not be able to see the edge of the widget. so I made it a button